### DISCUSSION TITLE
I don´t know how can I show on a grid screen the name of supplier insted of the id

### DISCUSSION THREAD

**Discussion by marcel-nasser** [Budibase/budibase#13742] [Help] [answered]
I have a screen based in a table named Product.
This table has the follow columns: productcode, productname, productvalue and a foreing key to Product table.
The screen has a grid and a crud generate automatically by Budibase.
The problem is how can I show the name of supplyer insted of id supplyer in grid and form.

**Comment by mikesealey** [comment, marked as answer]
Hi @marcel-nasser 

I think this is a fairly straightforward issue - in the table you're linking to, go to the colum you want to display instead of the ID, click the kebab menu, and click "Use as display column"
<img width="557" alt="image" src="https://github.com/Budibase/budibase/assets/39269380/aa066ca0-c57d-4334-8e73-80d8fae29c49">

Hope this helps.
